src/main/util.ts

```diff
@@ -452,6 +452,26 @@ const cutVideo = async (initialFile: string, finalDir: string, desiredDuration:
     });
 }
 
+/**
+ * Gets string value from the config in a more reliable manner.
+ * @param cfg the config store
+ * @param key the key
+ * @returns the string config
+ */
+const getPathConfigSafe = (cfg: any, key: string) : string => {
+  return cfg.has(key) ? path.join(cfg.get(key), "/") : "";
```

Comment:
As sorry I think I should have been clearer. I don't like that we have to add the "/" at all here, we should just use path methods everywhere when we are handling paths. I'm going to revert this so it can go in and I'll probably fix it another time.

Reply:
Reasoning is it's hard from a maintainability perspective if we need to remember if a path string has a "/" on the end or not. If we just use path.join() everywhere we don't need to worry. I just didn't know about path.join at the time of writing.

Reply:
So this is a little tricky because if we dont have this the way it is and then we get from the store when there isnt anything there (first time user), this will result in a string of "/", which gets interpreted as a non-string and is the reason we were throwing an error before. the way i have this now resolves this and gives us a good way to get Paths. 

we can discuss alternate solutions if you dont liek this one.